frontend/src/components/common/table/table.module.scss

```diff
@@ -0,0 +1,37 @@
+.clientSideTable {
+  border-collapse: collapse;
+}
+
+.tableHeaderCell,
+.tableCell {
+  padding: 13px 100px 13px 13px;
+  font-weight: lighter;
+  font-size: 16px;
+  font-style: normal;
+  line-height: 20px;
+  text-align: left;
+  vertical-align: center;
+}
+
+.tableHeaderRow {
+  background-color: #393939;
+  border-bottom: 2px solid #282828;
+}
+
+.tableHeaderCell {
+  color: rgba(255 255 255 / 69%);
+}
+
+.tableRow {
+  background-color: #393939;
+}
+
+.tableRow:hover {
+  background-color: #2a2a2a82;
+  cursor: pointer;
+  transition-duration: 500ms;
+}
+
+.tableCell {
+  color: #ffffff;
+}
```

Comment:
I think its better to use styles nesting here for visibility

Reply:
Eslint: Expected nesting length to be no more than 0